src/main/java/org/folio/rest/impl/PutOrderLineByIdHelper.java

```diff
@@ -49,8 +50,8 @@ public PutOrderLineByIdHelper(Map<String, String> okapiHeaders, Handler<AsyncRes
   /**
    * Handles update of the order line. First retrieve the PO line from storage and depending on its content handle passed PO line.
    */
-  public void updateOrderLine(String orderId, PoLine compOrderLine) {
-    getPoLineByIdAndValidate(orderId, compOrderLine.getId())
+  public void updateOrderLine(PoLine compOrderLine) {
+    getPoLineByIdAndValidate(compOrderLine.getId())
```

Comment:
The case when `PoLine compOrderLine` does not contain PO id is not covered seems

Reply:
added checking for `orderId `in `PoLine`